src/SkeletalTrapezoidationEdge.h

```diff
@@ -39,11 +70,53 @@ class SkeletalTrapezoidationEdge
     {
         return is_marked >= 0;
     }
+
+    bool hasTransitions(bool ignore_empty = false) const
+    {
+        return transitions_.use_count() > 0 && (ignore_empty || ! transitions_.lock()->empty());
+    }
+    void initTransitions(std::shared_ptr<std::list<TransitionMiddle>> storage)
+    {
+        transitions_ = storage;
+    }
+    std::shared_ptr<std::list<TransitionMiddle>> transitions()
+    {
+        return transitions_.lock();
+    }
+
+    bool hasTransitionEnds(bool ignore_empty = false) const
+    {
+        return transition_ends_.use_count() > 0 && (ignore_empty || ! transition_ends_.lock()->empty());
+    }
+    void initTransitionEnds(std::shared_ptr<std::list<TransitionEnd>> storage)
+    {
+        transition_ends_ = storage;
+    }
+    std::shared_ptr<std::list<TransitionEnd>> transition_ends()
+    {
+        return transition_ends_.lock();
+    }
+
+    bool hasExtrusionJunctions(bool ignore_empty = false) const
+    {
+        return extrusion_juntions_.use_count() > 0 && (ignore_empty || ! extrusion_juntions_.lock()->empty());
+    }
+    void initExtrusionJunctions(std::shared_ptr<std::vector<ExtrusionJunction>> storage)
+    {
+        extrusion_juntions_ = storage;
+    }
+    std::shared_ptr<std::vector<ExtrusionJunction>> extrusion_juntions()
```

Comment:
Shouldn't this be extrusionJunctions() or getExtrusionJunctions() (Or well at least extrusion_junctions, because I think the missing c should be added :P)

Reply:
Probably, and yes.